### PR TITLE
Fix: Staking card stack

### LINKED ISSUE #2767: Use new CardStack for staking ShardedAccount
Right now ShardedAccount on Staking page looks not how it should according to design.
Use new UI CardStack component developed for PotfolioView.

### PR DESCRIPTION
**Pull request by tuul-wq**
- `CardStack` for sharded list
- Updated visuals for list items

closes #2767 

image

### CODE REVIEW

File: src/renderer/shared/ui-kit/Checkbox/Checkbox.tsx [hunk 1]
```diff
@@ -21,6 +22,7 @@ export const Checkbox = ({
   checkboxPosition = 'center',
   children,
   onChange,
+  onClick,
```
Comment (tuul-wq) [inline src/renderer/shared/ui-kit/Checkbox/Checkbox.tsx:25]: Needed to somehow stop propagation of checkbox event

**Review by sokolova-an — APPROVED** [review]

File: src/renderer/shared/ui/Markdown/Markdown.tsx [hunk 2]
```diff
@@ -72,17 +73,16 @@ const components: Components = {
       <Checkbox
         {...props}
         onChange={(newCheckedState: boolean) => {
-          if (typeof props.onChange === 'function') {
```
Comment (johnthecat) [inline src/renderer/shared/ui/Markdown/Markdown.tsx:75]: Why do you need this change?

File: pnpm-lock.yaml
Comment (johnthecat) [inline pnpm-lock.yaml:1]: Why it was changed?

**Review by johnthecat — CHANGES_REQUESTED** [review]

File: src/renderer/shared/ui/Markdown/Markdown.tsx (hunk 2 above)
Comment (tuul-wq) [reply src/renderer/shared/ui/Markdown/Markdown.tsx:75]: fast `return` leads to less nesting

**Review by Asmadek — APPROVED** [review]

File: src/renderer/shared/ui/Markdown/Markdown.tsx (hunk 2 above)
Comment (johnthecat) [reply src/renderer/shared/ui/Markdown/Markdown.tsx:75]: Is it necessary in context of this PR?

**Review by johnthecat — APPROVED** [review]

File: src/renderer/shared/ui/Markdown/Markdown.tsx (hunk 2 above)
Comment (tuul-wq) [reply src/renderer/shared/ui/Markdown/Markdown.tsx:75]: Just a little 😉
For me it's quite natural to improve readability with minor rearrangements 
If you're strongly against this practice, I will hold myself

**Pull request merged**